target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-r619ac-128m.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-r619ac.dtsi"
+
+/ {
+	model = "P&W R619AC 128M";
+	compatible = "p2w,r619ac-128m";
+};
+
+&nand_rootfs {
+	reg = <0x0 0x8000000>;
```

Comment:
A comment about the bootloader's 64MiB limit would be very thoughtful. 

(At this point, I prefer if the whole nand node was copied, even if just the size in the reg property is different.
But that's just me, you don't need to change it.)

Reply:
Seconded, I agree that there should be a comment mentioning the stock MIBIB being set up for a 64MB chip,